tests/functional/pro/public/Standard_Source_By_Default_Cest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace ISC\Tests\Functional;
+
+/**
+ * Test ISC_Pro_Public::use_standard_source_by_default
+ */
+class Standard_Source_By_Default_Cest {
+
+	public function _before(\FunctionalTester $I) {
+		// enable the overlay
+		$existingOption = $I->grabOptionFromDatabase('isc_options');
+		$existingOption['display_type'] = ['overlay'];
+		// Make sure we enable the "custom_text" standard source with the text "@ ISC"
+		$existingOption['standard_source'] = 'custom_text';
+		$existingOption['standard_source_text'] = '@ ISC';
+		// Show the standard source if the image’s specific source is empty
+		$existingOption['use_standard_source_by_default'] = 1;
+
+		// Update the option in the database with the new array.
+		$I->haveOptionInDatabase( 'isc_options', $existingOption );
+
+		// prepare an image
+		$I->havePostInDatabase( [
+           'ID' => 123,
+           'post_title' => 'Image One',
+           'guid' => 'https://example.com/image-one.jpg',
+       ] );
+		// image has a source by default
+		$I->havePostmetaInDatabase( 123, 'isc_image_source', 'Author A' );
+	}
```

Comment:
**Suggestion:** Consider using a setup method to initialize common test data and reduce redundancy. This will make your tests more maintainable and easier to read. [maintainability]
```suggestion
 protected function setUp(): void
 {
     parent::setUp();
     $this->I = $this->getModule('\Helper\Functional');
     $this->initializeTestData();
 }

 protected function initializeTestData()
 {
     // enable the overlay
     $existingOption = $this->I->grabOptionFromDatabase('isc_options');
     $existingOption['display_type'] = ['overlay'];
     // Make sure we enable the "custom_text" standard source with the text "@ ISC"
     $existingOption['standard_source'] = 'custom_text';
     $existingOption['standard_source_text'] = '@ ISC';
     // Show the standard source if the image’s specific source is empty
     $existingOption['use_standard_source_by_default'] = 1;

     // Update the option in the database with the new array.
     $this->I->haveOptionInDatabase( 'isc_options', $existingOption );

     // prepare an image
     $this->I->havePostInDatabase( [
        'ID' => 123,
        'post_title' => 'Image One',
        'guid' => 'https://example.com/image-one.jpg',
    ] );
     // image has a source by default
     $this->I->havePostmetaInDatabase( 123, 'isc_image_source', 'Author A' );
 }
```